gui-apps/xwaylandvideobridge/xwaylandvideobridge-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.83.0
+QTMIN=5.15.0
+KDE_ORG_CATEGORY="system"
+
+inherit ecm kde.org xdg
```

Comment:
you're not using xdg here, right?

Reply:
It installs an icon and desktop file, so I inherit `xdg` to update those databases in postinst and postrm.

Reply:
`ecm.eclass` is already doing the magic for you.

Reply:
> `ecm.eclass` is already doing the magic for you.

Great! I dropped the explicit inherit of `xdg`.